website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -119,7 +120,33 @@ _With a VCS repository_
       "vcs-repo": {
         "identifier": "example/terraform-test-proj",
         "oauth-token-id": "ot-hmAyP66qk2AMVdbJ",
-        "branch": ""
+        "branch": "",
+        "tags-regex": null
+      },
+      "updated-at": "2017-11-29T19:18:09.976Z"
+    },
+    "type": "workspaces"
+  }
+}
+```
+_Using Git Tags
+
+A run will be triggered when a Git tag is pushed matching the regular expression (SemVer): `1.2.3`, `22.33.44`, etc.
```

Comment:
I suggest rewording to "A pushed Git tag matching the regular expression triggers a run."

Reply:
I've noticed we're changing narrative in few fields to include first-person view - e.g. `when you push`, `your repository` etc. Is this going to be a preferred path forward?

Applied @laurapacilio suggestion -> be9cee61269a3550f30371f5263f035e121841f1 -> but also left the example from before for the clarification